src/traces/image/calc.js

```diff
@@ -0,0 +1,35 @@
+/**
+* Copyright 2012-2019, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+// var Registry = require('../../registry');
+// var Lib = require('../../lib');
```

Comment:
:zap: those commented lines please

Reply:
Done in 73c0740f53fc82a70196d1993ba2824db2be85a3